libs/ledger/src/tx_status_http_interface.cpp

```diff
@@ -85,7 +85,7 @@ constexpr PublicTxStatus Convert(TransactionStatus       tx_processing_pipeline_
     case ContractExecutionStatus::RESOURCE_FAILURE:
     case ContractExecutionStatus::TX_LOOKUP_FAILURE:
     case ContractExecutionStatus::INEXPLICABLE_FAILURE:
```

Comment:
I'd find easier to understand the flow if there was a default after these, instead of a `break` and the equivalent of a default outside the `switch`.

Reply:
We avoid using `default` in switch statements on enums - the presence of `default` prevents most compilers from warning us if there's a value we didn't cover. It's more explicit and therefore less error-prone to be forced to enumerate all the enum values, even if some of them are no-ops.